tests/native_tests/language.hy

```diff
@@ -266,6 +266,25 @@
   (assert (= count 0))
   (assert (= fact 120)))
 
+(defn test-while-loop-else []
+  "NATIVE: test while loops?"
```

Comment:
Why does this docstring end with a question mark? (To be fair, the docstrings of tests no longer have much effect since we switched from Nose to pytest. You can just not use one at all if you want.)

Reply:
Only because https://github.com/hylang/hy/blob/51fb807cc9eb5b005b79b4dbe819d8e11889de78/tests/native_tests/language.hy#L260 does - I'll remove.